src/praisonai/praisonai/cli/features/onboard.py

```
@@ -494,35 +494,66 @@ def _mask(value: str) -> str:
                 result = install_daemon(config_path=self.config_path)
                 if result.get("ok"):
-                    console.print(f"  [green]✓[/green] {result.get('message', 'Service installed')}")
+                    console.print(
+                        f"  [green]✓[/green] {result.get('message', 'Service installed')}"
+                    )
                 else:
-                    console.print(f"  [red]✗[/red] {result.get('error', 'Install failed')}")
+                    console.print(
+                        f"  [red]✗[/red] {result.get('error', 'Install failed')}"
+                    )
             except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)